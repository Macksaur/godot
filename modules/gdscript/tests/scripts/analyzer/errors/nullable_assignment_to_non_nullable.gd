func test():
    var foo: Array[int]? = [1, 2]
    var bar: Array[int] = foo