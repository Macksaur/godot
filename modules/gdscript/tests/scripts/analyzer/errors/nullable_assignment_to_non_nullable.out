GDTEST_ANALYZER_ERROR
Potentially unsafe to assign nullable "foo" to non-nullable "bar".
